src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -1088,7 +1088,12 @@ public async Task<int> StartNewReleaseAsync(string accountName, string projectNa
         /// <param name="queueTimeVariables">A string in JSON format containing the queue time variables to be used.</param>
         public async Task<int> StartNewBuildAsync(string accountName, string projectName, int azdoDefinitionId, string sourceBranch, string sourceVersion, string queueTimeVariables = null)
         {
-            var body = $"{{ \"definition\": {{ \"id\": \"{azdoDefinitionId}\" }}, \"sourceBranch\": \"{sourceBranch}\", \"sourceVersion\": \"{sourceVersion}\", \"parameters\": '{queueTimeVariables}'  }}";
+            var body = $"{{ \"definition\": " +
```

Comment:
[nit] perhaps usin a n $@ strign is better here?

Reply:
Why not just `new JObject(<stuff>).ToString()`?